packages/ad/README.md

```diff
@@ -45,6 +45,7 @@ import { ZeusAdWithRouter } from '@zeus-technology/ad';
 
 const App = (props) => (
```

Comment:
We should add docs here to say that if you use this component you _must_ use the `ZeusRouteResponser`.